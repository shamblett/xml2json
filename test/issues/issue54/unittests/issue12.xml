<?xml version="1.0" encoding="utf-8"?>
<zoo>
  <fish>
    <name>whale</name>
    <weight/>
  </fish>
  <bird>
    <name>owl</name>
  </bird>
</zoo>
