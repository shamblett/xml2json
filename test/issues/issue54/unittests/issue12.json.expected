{"zoo":{"fish":{"name":"whale","weight":null},"bird":{"name":"owl"}}}
